src/Statement/MysqlStatementBuilder.php

```diff
@@ -271,6 +274,26 @@ public function buildFunction(FunctionOperation $operation): string
         }
     }
 
+    /**
+     * Builds SQL statement for an execute operation.
+     *
+     * @param ExecOperation $operation
+     * @return string
+     * @throws UnsupportedOperationException
+     */
+    public function buildExecute(ExecOperation $operation): string
+    {
+        switch ($operation->getOperation()) {
+            case ExecOperation::EXEC:
+                return sprintf(
```

Comment:
The `sprintf` and use of a constant is redundant here. The result of `$operation-getBody()` may be returned directly instead.

Reply:
Gotcha, handled!